microsoft/PowerToys
# Peek: icons, removed unneeded RTL code, ui tweaks and code suggestions

## Summary of the Pull Request

- Changed icon codes for the Pin button
- TitleBarHeightOption.Tall
- Removed a lot of unneeded RTL code
- Small tweaks in the Previewers (all subtle)
- Lots of suggestions by Visual Studio / Analyzer (functionally irrelevant)

## PR Checklist

- [x] Closes: #26134, #27261

## Validation Steps Performed

- Tested with Windows Language = Hebrew (RTL layout, which isn't applied by the Peek **window**, but content is alright)
- Double-checked light/dark theme switching and scaling

![image](https://github.com/microsoft/PowerToys/assets/65828559/c82c3866-34c5-4d73-9415-d88e1ecc1c02)
> ↳ Drive, default size

![image](https://github.com/microsoft/PowerToys/assets/65828559/1abca1bb-1a46-4358-8d32-7b34933c46da)
> ↳ Folder, minimum size

https://github.com/microsoft/PowerToys/assets/65828559/f3620acb-68cb-4a2b-8130-3ad60916d5b2
> ↳ Scrolling back and forth between 3 files (notice the bug)

![image](https://github.com/microsoft/PowerToys/assets/65828559/db011288-24a5-4270-ac92-e449dd11eb2d)
> ↳ RTL OS (notice the window title bar doesn't change)

FYI @SamChaps
